src/paradox.c

```diff
@@ -3857,6 +3857,7 @@ PX_put_data_alpha(pxdoc_t *pxdoc, char *data, int len, char *value) {
 #else
 #if PX_USE_ICONV
 		size_t ilen = strlen(value);
+		size_t res;
```

Comment:
If this was needed for building on windows and when inside the #if, why is in the same #if above, the type is different?
https://github.com/DRSDavidSoft/pxlib/blob/cbbd401b0da7d7068c69c9cc42f46c9d4fc262ff/src/paradox.c#L3422